CHANGELOG.rst

```diff
@@ -12,18 +12,28 @@ Internal Changes
 
 * **Gateway**: Add endpoint for refreshing expiring Gitlab tokens
 * **Sessions**: Check LFS size and available disk space before cloning
+* **Knowledge Graph**: Improve access tokens management
+* **Knowledge Graph**: Add migration importing Composite Plans into the Triples Store
```

Comment:
Is this a new migration?

How about?
Add support for migrating Composite Plans into Triples Store

Reply:
Actually, the change is not about adding support for migration. The change is a migration that goes through the Triples Store (TS) seeking Projects that do not have CompositePlan entities while they should. For these, we raise events causing the transformation process to redo relevant commit events and made the missing CompositePlans added to the TS. What about:

Reply:
Maybe: `Enhance migration functionality to add missing CompositePlans when necessary`